community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -742,7 +756,7 @@ public SchemaIndexProviderMap schemaIndexProviderMap()
         };
     }
 
-    private StoreLayerModule buildStoreLayer( NeoStores neoStores,
+    private StoreLayerModule buildStoreLayer( final NeoStores neoStores,
```

Comment:
This `final` annotation is not needed, since you extracted the store statement factory creating into its own method.